### PR TITLE
jihad c4 fixes

### PR DESCRIPTION
**Pull request by P250**
- If player presses E to detonate bomb but dies before 1s timer, the bomb doesn't explode.
- People should be able to reliably detonate the bomb now...

### CODE REVIEW

File: mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs [hunk 1]
```diff
@@ -82,6 +86,7 @@ private void PlayerUseC4ListenerCallback()
     [GameEventHandler]
     public HookResult OnPlayerDropC4(EventBombDropped @event, GameEventInfo info)
     {
+        Console.WriteLine("2");
```
Comment (jithatsonei) [inline mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs:89]: remove

File: mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs [hunk 2]
```diff
@@ -167,34 +170,32 @@ public HookResult OnPlayerLeave(EventPlayerDisconnect @event, GameEventInfo info
     /// <param name="player">The player who should receive the Jihad C4.</param>
     public void TryGiveC4ToPlayer(CCSPlayerController player)
     {
+        Console.WriteLine("1");
```
Comment (jithatsonei) [inline mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs:173]: remove

File: mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs [hunk 3]
```diff
@@ -10,13 +10,14 @@
 using Jailbreak.Public.Extensions;
 using Jailbreak.Public.Mod.Rebel;
 using Microsoft.Extensions.Logging;
+using System.Reflection.Metadata.Ecma335;
```
Comment (P250) [inline mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs:13]: idk what this is

**Review by jithatsonei — CHANGES_REQUESTED** [review]

File: mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs (hunk 3 above)
Comment (jithatsonei) [reply mod/Jailbreak.Rebel/JihadC4/JihadC4Behavior.cs:13]: remove it

**Review by jithatsonei — APPROVED** [review]

**Pull request merged**